deepmd/pt/infer/deep_eval.py

```
@@ -360,9 +367,19 @@ def eval(
+                cells,
+                atom_types,
+                fparam,
+                aparam,
+                atomic_weight,
+                request_defs,
             )
         else:
+            if atomic_weight is not None:
+                raise ValueError(
+                    "atomic_weight is not supported when spin is provided. "
+                )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)